posts/2023-06-21-data-protection-compliance-with-distributed-ledger-erasure.md

```diff
@@ -0,0 +1,241 @@
+[California Consumer Privacy Act](https://oag.ca.gov/privacy/ccpa) (CCPA) in California provide a legal framework to ensure personal information is appropriately protected.
+
+Maintaining compliance with these regulations is often non-trivial and places a significant burden on organizations in control
+of personal information.
+
+The Vendia Share platform was fundamentally designed around data protection principles. Requirements such
+as [access control](https://www.vendia.com/blog/data-protection-approaches) and [permanent data erasure](https://www.vendia.com/docs/share/erasure) 
+are built natively into the platform and distributed ledger.
+
+This enables Vendia Share to provide a tamper-proof and verifiable distributed ledger 
+while also enabling data controllers to easily stay in compliance with data protection regulations.
+
+This article will discuss how Vendia Share's data erasure functionality is built in to the platform and how it can be used
+to stay in compliance with data erasure regulations.
+
+## Right to erasure
+
+Both GDPR and CCPA include "right to erasure" requirements. That is, users can request to have personal information permanently erased
+without undue delay.
+
+[GDPR Article 17](https://gdpr-info.eu/art-17-gdpr/) states that "The data subject shall have the right 
+to obtain from the controller the erasure of personal data concerning him or her without undue delay and the controller shall have the obligation to erase personal data without undue delay..."
+
+In CCPA, [section 1798.105](https://leginfo.legislature.ca.gov/faces/codes_displaySection.xhtml?lawCode=CIV&sectionNum=1798.105) states
+that "A consumer shall have the right to request that a business delete any personal information about the consumer which the business has collected from the consumer"
+
+Organizations acting as data controllers and using third party platforms as data processors must consider the platforms
+ability to support timely and permanent data erasure in order to maintain compliance with these regulations.
+
+## Data erasure in distributed ledgers
+
+In many traditional systems, permanent data erasure is trivial. It is often as simple as issuing a SQL query to delete rows
+or columns from a transactional database, or deleting a file from a file system or object store.
+
+However, in distributed ledger systems the concept of permanent and irrevocable erasure is fundamentally
+at odds with the design of such systems. Ledgers are by nature immutable. 
+
+How can organizations stay in compliance when storing personal information in an immutable, append-only ledger?
+
+A typical approach often involves client-side "crypto-shredding" where all personal information is encrypted
+using an encryption key on the client before it is persisted to the ledger. Then if a user requests for information to be deleted, 
+the data controller simply needs to delete the encryption key.
```

Comment:
This question is for myself, so in our case, we are using an encryption key for each of our field? and when things are erased, the content isn't changed, but we toss out the encryption key for that field?

Reply:
correct